conn.go

```diff
@@ -260,10 +260,14 @@ func (c *Conn) clearPendingRequests() {
 }
 
 func (c *Conn) getXid() int32 {
```

Comment:
I know it's kinda not related to the diff but it's not very common to use "getters" in Go :) Just `Xid()` should be sufficient :)

Reply:
May I suggest `nextXid()` to make it clear it's stateful.